cfgov/sheerlike/templates.py

```diff
@@ -2,12 +2,25 @@
 from dateutil import parser
 from pytz import timezone
 
-def date_formatter(value, format="%Y-%m-%d", tz='America/New_York'):
+
+def date_filter(value, format="%Y-%m-%d", tz='America/New_York'):
```

Comment:
Probably but I didn't want to break anything that was using this since this PR doesn't use it. At some point (during a refactor) this will probably get deprecated.